BrainPortal/app/models/cluster_task.rb

```diff
@@ -2416,13 +2419,15 @@ def singularity_commands(command_script)
 
 # Invoke Singularity with our wrapper script above.
 # Tricks used here:
-# 1) we mount the gridshare root directory
-# 2) we mount the local data provider cache root directory
-# 3) we mount each (if any) of the root directory for local data providers
-# 4) we mount (if any) file system overlays
-# 5) with -H we set the task's work directory as the singularity $HOME directory
+# 1) we supply (if any) additional options for the exec command 
+# 2) we mount the gridshare root directory
+# 3) we mount the local data provider cache root directory
+# 4) we mount each (if any) of the root directory for local data providers
+# 5) we mount (if any) file system overlays
+# 6) with -H we set the task's work directory as the singularity $HOME directory
 #{singularity_executable_name}                  \\
     exec                                        \\
+    #{container_params || '            '}       \\
```

Comment:
No need for the `|| '   '`  it make the code more difficult to read. nil can be substituted in a string perfectly fine (thus the `.presence` in my other comment). And there's no need to try to align the spacing, it's already super wobbly with the other arguments